### ISSUE TITLE
Tab width is bigger than size that expected.

### ISSUE THREAD

**Issue by shayanpourvatan** [roughike/BottomBar#626]
Hi, first of all thanks for your great library

I have an issue with tab size in Samsung GT-I9070 android 4.1.2
after reading your code i figure out that your code for converting `pixelToDp` has some issues, The issues will be solved after changing `pixelToDp` function like below: 

```
protected static int pixelToDp(Context context, int px) {
        Resources resources = context.getResources();
        DisplayMetrics metrics = resources.getDisplayMetrics();
        float dp = px / ((float) metrics.densityDpi / DisplayMetrics.DENSITY_DEFAULT);
        return (int) dp;
}
```
please consider this issue in your next release and let me know when you will fix the issue!

**Comment by roughike** [comment]
This should be fixed as of ```2.1.1```. If you can, please confirm if it works for you too.

**Comment by shayanpourvatan** [comment]
thanks @roughike it's working.
